projects/sport-data-valley/seeds/questionnaires/mmm_questionnaire.rb

```diff
@@ -0,0 +1,508 @@
+# frozen_string_literal: true
+
+MMM_QUESTIONNAIRE_NAME = 'move_mood_motivation'
+
+db_name = MMM_QUESTIONNAIRE_NAME
+questionnaire = Questionnaire.find_by(name: db_name)
+questionnaire ||= Questionnaire.new(name: db_name)
+questionnaire.key = db_name
+
+frequency_options = [
+  {
+    title: {  nl: "Nooit",
+            en: "Never" },
+    value: "never",
+    numeric_value: 1
+  },
+  {
+    title: {  nl: "Zelden",
+            en: "Rarely" },
+    value: "rarely",
+    numeric_value: 2
+  },
+  {
+    title: {  nl: "Soms",
+            en: "Sometimes" },
+    value: "sometimes",
+    numeric_value: 3
+  },
+  {
+    title: {  nl: "Vaak",
+            en: "Often" },
+    value: "often",
+    numeric_value: 4
+  },
+  {
+    title: {  nl: "Altijd",
+            en: "Always" },
+    value: "always",
+    numeric_value: 5
+  },
+]
+
+agreement_options = [
+  {
+    title: {  nl: "Zeer oneens",
+            en: "Strongly disagree" },
+    value: "strongly-disagree",
+    numeric_value: 1
+  },
+  {
+    title: {  nl: "Oneens",
+            en: "Disagree" },
+    value: "disagree",
+    numeric_value: 2
+  },
+  {
+    title: {  nl: "Neutraal",
+            en: "Neutral" },
+    value: "neutral",
+    numeric_value: 3
+  },
+  {
+    title: {  nl: "Eens",
+            en: "Agree" },
+    value: "agree",
+    numeric_value: 4
+  },
+  {
+    title: {  nl: "Zeer eens",
+            en: "Strongly agree" },
+    value: "strongly-agree",
+    numeric_value: 5
+  },
+]
+
+
+question_content = [
+  {
+    type: :raw,
+    content: { nl: "<h4 class=\"header\">Move, Mood & Motivation vragenlijst</h4>",
+                en: "<h4 class=\"header\">Move, Mood & Motivation questionnaire</h4>" }
+  },
+  {
+    id: 'activity_tracker',
```

Comment:
Are you sure you'd not stick with out v1, v2,  etc. convention for question names? Up to you of course, but it's like this in every other questionnaire.

Reply:
I know it goes against convention, but I think it is better because the current convention has a problem: both the developers and users of JLab have problems understanding what the values are when they retrieve the data because all of them are named "vX". So, you always have to open the questionnaire/questionnaire definition to understand what you are talking about.
Making the question ids something readable makes it less prone to errors and speeds up the process of understanding what is in the data.